src/line_detect/test/test_utilities/line_detect_test_utils.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Created By: Cameron Newton
+ * Created On: June 16th, 2018
+ * Description: Utilities to aid in debugging GateDetector module
+ */
+
+#include "../../include/LineDetector.h"
```

Comment:
Should just be `#include "LineDetector.h"`

Reply:
fixed